### PR TITLE
[Brent] [Waste] Add garden subscription notice.

### PR DESCRIPTION
**Pull request by neprune**
[skip changelog]

https://mysocietysupport.freshdesk.com/a/tickets/4187

brent subscription text

### CODE REVIEW

**Review by dracos — CHANGES_REQUESTED** [review]
If they're already subscribed, there's no way they could view this page, which is the intro page for someone who doesn't have a subscription :)
They didn't say where they wanted this, but from what I read I thought they wanted it on the bin day page - where if you're staff it prints out "Modify your garden subscription" it would have the text there, alongside their garden waste subscription information. 

Given they must already have a sub to view that, the text could be "If you are looking to add another service, please contact our customer services team on 020 8937 5050 who will be able to do this for you."

**Comment by neprune** [comment]
> If they're already subscribed, there's no way they could view this page, which is the intro page for someone who doesn't have a subscription :) They didn't say where they wanted this, but from what I read I thought they wanted it on the bin day page - where if you're staff it prints out "Modify your garden subscription" it would have the text there, alongside their garden waste subscription information.
> 
> Given they must already have a sub to view that, the text could be "If you are looking to add another service, please contact our customer services team on 020 8937 5050 who will be able to do this for you."

Well that makes sense 😅 . Thanks 🙏

**Comment by neprune** [comment]
demo screenshot

File: templates/web/base/waste/services.html [hunk 1]
```diff
@@ -47,6 +47,11 @@
          [% IF c.cobrand.moniker == 'kingston' AND (slwp_garden_sacks AND unit.garden_sacks) %]
              <input type="submit" value="Order more garden sacks" class="waste-service-descriptor waste-service-link">
          [% ELSE %]
+            [% IF c.cobrand.moniker == 'brent' %]
+                <p>
+                  If you already have an active Garden Waste subscription and are looking to add another service, please contact our customer services team on 020 8937 5050 who will be able to do this for you.
```
Comment (dracos) [inline templates/web/base/waste/services.html:52]: To get here, you must have a subscription, so the first part of this If isn't needed, can just be "If you are looking to add..."

File: templates/web/base/waste/services.html [hunk 2]
```diff
@@ -47,6 +47,11 @@
          [% IF c.cobrand.moniker == 'kingston' AND (slwp_garden_sacks AND unit.garden_sacks) %]
              <input type="submit" value="Order more garden sacks" class="waste-service-descriptor waste-service-link">
          [% ELSE %]
+            [% IF c.cobrand.moniker == 'brent' %]
```
Comment (dracos) [inline templates/web/base/waste/services.html:50]: This is only reached if `show_garden_modify` is true, which for Brent is only for staff (that's why we want to show this message), so this isn't shown to normal users (see normal user display attached). So it needs to be its own section, presumably can still be inside the `NOT pending_cancellation` section, but outside this section.

image

**Review by dracos — CHANGES_REQUESTED** [review]
Sorry, afraid this still won't appear where you want it to

**Comment by neprune** [comment]
@dracos sorry, not my brightest moment, have changed to appear for non-staff

**Review by dracos — APPROVED** [review]
Looks good, thanks :)

**Pull request merged**